.env.example

```diff
@@ -0,0 +1,3 @@
+LOCALSTACK_AUTH_TOKEN=YOUR_TOKEN
+AWS_DEFAULT_REGION=us-east-1
+# LOCAL_RUN=false # Default is true
```

Comment:
Q: The presence of this environment variable leads me to the impression that the `terraform` stack can already be deployed to AWS if I set this variable to `true`. 🤔 
In my opinion, this would be out of scope of this PR, but maybe we could still discuss the current state here? Does the terraform module already run against AWS? If not - and assuming you did try to run it against AWS - what do you think is left to do? 😄

Reply:
True, it is misleading. This terraform code was not tested against AWS, only against LocalStack. 

Some parts of the terraform code were wrtitten, so, they do not create unique resources in AWS (for example bucket names).

 I think to be deployable to AWS the following additional changes are needed:
1. Terraform state file must be stored somewhere. If we want to store it in an S3 bucket, we need to update README file and create a bucket for the terraform state. After it, refer to the bucket in the terraform code.
2. Code in the `website/app.js` must be modified, specifically logic around authorization for lambdas invocation.
3. Code in the  `website/index.html` refers to URL in LocalStack
4. SES notifications won't work because domain verification will be required